app/assets/stylesheets/components/_filter-summary.scss

```diff
@@ -0,0 +1,60 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-summary__remove-filters {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.app-c-filter-summary__remove-filter {
+  border: 1px solid $govuk-border-colour;
+  border-radius: govuk-em(5, 16px);
+  padding: govuk-spacing(2);
+  margin: 0 govuk-spacing(2) govuk-spacing(1) 0;
+  text-decoration: none;
+  display: inline-block;
+  color: $govuk-text-colour;
+  background-color: govuk-colour("light-grey");
+  @include govuk-font(16);
+
+  @include govuk-media-query($from: tablet) {
+    text-wrap: nowrap;
+  }
+
+  &:focus,
+  &:focus-visible {
+    background-color: $govuk-focus-colour;
+    outline: 2px $govuk-text-colour solid;
+    outline-offset: 0;
+    border-color: $govuk-text-colour;
+  }
+
+  &:hover {
+    outline: 1px $govuk-text-colour solid;
+    outline-offset: 0;
+    border-color: $govuk-text-colour;
+  }
+
+  &::before {
+    content: "\00d7";
+    vertical-align: middle;
+    height: govuk-spacing(2);
+    padding-right: 2px;
+    @include govuk-font(24);
+
+    @include govuk-media-query($from: tablet) {
+      line-height: 1;
+    }
+  }
+}
+
+.app-c-filter-summary__remove-filter-text {
+  vertical-align: middle;
+}
+
+.app-c-filter-summary__heading {
```

Comment:
Might make more sense for this to be further up so the order in the CSS follows that in the markup